components/brave_news/browser/resources/Feed.tsx

```diff
@@ -178,21 +186,21 @@ export default function Component({ feed, onSessionStart }: Props) {
         throw new Error("Invalid item!" + JSON.stringify(item))
       }
 
+      // Store the card count from before this element - it is the number of
+      // cards we've had to look at to reach this one.
+      const seenBeforeCard = currentCardCount;
```

Comment:
why not just use `currentCardCount` instead? (this question is coupled with the intersection observer question above)